zipline/data/minute_bars.py

```diff
@@ -269,12 +270,14 @@ def __init__(
         start_session,
         end_session,
         minutes_per_day,
+        version,
     ):
         self.calendar = calendar
         self.start_session = start_session
         self.end_session = end_session
         self.ohlc_ratio = ohlc_ratio
         self.minutes_per_day = minutes_per_day
+        self.version = version
```

Comment:
Aha, I see - the writer specifies the latest.  That makes sense now, though it was surprising to me that the caller passes that attribute of `BcolzMinuteBarMetadata` to its own constructor - maybe a default value would make sense.